src/commands/optional/uwurandom.rs

```diff
@@ -0,0 +1,38 @@
+use crate::Context;
+
+use color_eyre::eyre::Result;
+use log::debug;
+
+/// generate some amount of uwurandom
+#[poise::command(slash_command)]
+pub async fn uwurandom(
+	ctx: Context<'_>,
+	#[description = "the amount of uwurandom to generate"]
+	#[min = 1]
+	#[max = 2000]
+	length: u16,
```

Comment:
maybe this could be an optional argument and we send a (sensible) random amount of uwurandom by default? (peak randomness)